.yamllint.yml

```diff
@@ -14,3 +14,4 @@ rules:
   document-start: disable
   truthy:
     allowed-values: ['true', 'false', 'yes', 'no']
+    check-keys: false
```

Comment:
Given the low likelihood of specifying a value such as true for the key, I disabled the warning.